bouncer/src/repo/manipulator/modelconvertor/import/repo_model_import_3drepo.cpp

```diff
@@ -132,25 +137,76 @@ repo::core::model::MaterialNode *RepoModelImport::parseMaterial(const boost::pro
 	materials.insert(materialNode);
 	matNodeList.push_back(materialNode);
 
-	return materialNode;
+	if(textureId >= 0)
+	{
+		textureIdToParents[textureId].push_back(materialNode->getSharedID());
+	}
+	return;
+}
+
+void RepoModelImport::parseTexture(
+	const boost::property_tree::ptree& textureTree, 
+	char * const dataBuffer)
+{
+	bool fileNameOk	 = textureTree.find("filename") != textureTree.not_found();
```

Comment:
can we put these strings as constants somewhere? (top of file, top of function, a new file called `repo_model_import_3drepo_constants.h` , whatever makes the most sense)

I know this file itself doesn't seem to do this, but it 'd be a good habit to stop introducing magic strings everywhere, esp we reference it again on a different line

Reply:
good call, this is done.